### PR TITLE
LPS-151919 As a developer using headless endpoints, I'd like to update a region by regionId via partial update and full replace

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test:sf

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one inline comment.

File: modules/apps/headless/headless-admin-address/headless-admin-address-impl/src/main/java/com/liferay/headless/admin/address/internal/resource/v1_0/RegionResourceImpl.java [hunk 1]
```diff
@@ -122,6 +122,16 @@ public Region postRegion(Region region) throws Exception {
...
 	}
 
+	@Override
+	public Region putRegion(Long regionId, Region region) throws Exception {
+		com.liferay.portal.kernel.model.Region serviceBuilderRegion =
+			_regionService.updateRegion(
+				regionId, GetterUtil.get(region.getActive(), true),
+				region.getName(), region.getPosition(), region.getRegionCode());
+
+		return _toRegion(serviceBuilderRegion);
```
[the remark this answers is not in the document]
Comment (pei-jung) [reply modules/apps/headless/headless-admin-address/headless-admin-address-impl/src/main/java/com/liferay/headless/admin/address/internal/resource/v1_0/RegionResourceImpl.java:132]: Yes, SF allows it. I'll inline the var.

**Comment by epark18** [comment]
CI failures are unrelated.

**Comment by pei-jung** [comment]
see https://github.com/liferay-usm/liferay-portal/pull/567

**Pull request closed without merging**